symplyphysics/laws/hydro/hydrostatic_pressure_from_density_and_depth.py

```diff
@@ -0,0 +1,25 @@
+from sympy import Eq, solve
+from symplyphysics import units, Quantity, Symbol, print_expression, validate_input, validate_output
+
+# Description
+# Law: P = ρ * g * h
+# ρ - density of liquid
+# g - acceleration of gravity
+# h - depth
+
+density = Symbol('density', units.mass / units.volume)
+g = Symbol('g', units.acceleration)
+depth = Symbol('depth', units.length)
+hydrostatic_pressure = Symbol('hydrostatic_pressure', units.pressure)
+
+law = Eq(hydrostatic_pressure, density * g * depth)
+
+def print_law():
+    return print_expression(law)
+
+@validate_input(density_=density, depth_=depth, g_=g)
+@validate_output(hydrostatic_pressure)
+def calculate_hydrostatic_pressure(density_: Quantity, depth_: Quantity, g_: Quantity) -> Quantity:
```

Comment:
Remove `g` from parameters.

Reply:
sorry my bad, didnt notice PR, and i did commit. its a mistake u dont need to check it

Reply:
So should I review this PR or not? It looks pretty good for just discarding it.

Reply:
yep if u mean last commit, 3rd is mistake